### DISCUSSION TITLE
各位大佬们，我的pcl2打开就报错，有没有解决方案，错误图片在下面

### DISCUSSION THREAD

**Discussion by flounxisicoal** [Meloong-Git/PCL#2026] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/90231208/1b953da5-8a73-4886-869d-e9bca43b478d)

**Comment by Chiloven945** [comment]
~这种情况会有日志吗？~
有的话发一下

**Reply by flounxisicoal** [reply]
麻烦发一下日志的存储位置，是否是命名为log1.txt，感谢

**Reply by flounxisicoal** [reply]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/11714193/Log1.txt)

**Reply by Chiloven945** [reply]
> [Log1.txt](https://github.com/Hex-Dragon/PCL2/files/11714193/Log1.txt)

看了下是上次的，但是这就奇怪了你已经启动过了为什么会出现这种情况（）
你试着用楼下的方法吧

**Reply by flounxisicoal** [reply]
正在尝试系统重装

**Reply by flounxisicoal** [reply]
感谢你的帮助

**Comment by Pigeon0v0** [comment, marked as answer]
应该是系统不完整

尝试前往 设置 - 系统 - 恢复 重置此电脑

**Reply by flounxisicoal** [reply]
感谢提供的建议，现已恢复正常
